examples/jupyter/README.md

```diff
@@ -0,0 +1,17 @@
+# Jupyter 
+It is rather simple to deploy Jupyter on MANTL. There are some official Docker images that you can use for this purpose (https://github.com/jupyter/docker-stacks). For this example we use the jupyter/minimal-notebook.
+
+
+We wrapped the Marathon submit REST call in a small script: `deploy.sh`. You can use it to deploy Jupyter on your MANTL cluster.
+
+```bash
+./deploy.sh
+```
+
+If everything went fine, you should be able to figure out the front end URL of your Jupyter deployment from the Traefik UI.
```

Comment:
Yes, but in Marathon you get only the back end URL. You can use that only if you are deploying locally (or if you opened all of the ports in your resource nodes). I'll write a bit about that.